src/WebJobs.Script/Workers/Rpc/Configuration/RpcWorkerConfigFactory.cs

```diff
@@ -40,6 +41,7 @@ public RpcWorkerConfigFactory(IConfiguration config, ILogger logger, ISystemRunt
             {
                 WorkersDirPath = workersDirectorySection.Value;
             }
+            _environment.SetEnvironmentVariable(EnvironmentSettingNames.WorkerDirectoryPath, WorkersDirPath);
```

Comment:
Why is this being set? Seems outside of the scope of this change.

Reply:
We can create a new PR just for this change, but it's semi profiles related; essentially this is for the workers to be able to get the worker path through something like  `%WORKER_PATH%` in the arguments property in the worker.config file etc.